rmw_connext_shared_cpp/include/rmw_connext_shared_cpp/event.hpp

```diff
@@ -22,6 +22,26 @@
 
 #include "rmw_connext_shared_cpp/visibility_control.h"
 
+/// Initialize a rmw_event_t.
+/**
+ * \param[in|out] rmw_event to initialize
+ * \param topic_endpoint_impl_identifier implementation identifier of event's parent topic endpoint
+ * \param data to initialize with
+ * \param event_type for the event to handle
+ * \return `RMW_RET_OK` if successful, or
+ * \return `RMW_RET_INVALID_ARGUMENT` if invalid argument, or
+ * \return `RMW_RET_UNSUPPORTED` if event_type is not supported, or
+ * \return `RMW_RET_ERROR` if an unexpected error occurs.
+ */
+RMW_CONNEXT_SHARED_CPP_PUBLIC
+rmw_ret_t
+__rmw_init_event(
```

Comment:
A bit unrelated because it's extensively used in this package, function names starting with a leading underscore or containing `__` should be avoided, as those names are reserved by the standard.
See https://github.com/ros2/rcl/pull/597#discussion_r392422395.

Reply:
We can do a separate pull request that changes all the function names in this repo consistently.